src/components/checkout/payment-methods.js

```diff
@@ -0,0 +1,62 @@
+// Libraries
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+// Lib
+import componentMapping from '../../lib/component-mapping'
+
+class PaymentMethods extends Component {
+  constructor (props) {
+    super(props)
+    this.Button = componentMapping('Button')
+    this.PaymentMethodHeader = componentMapping('PaymentMethodHeader')
+    this.PayPalButton = componentMapping('PayPalButton')
+  }
+
+  /**
+   * When a customer selects the default checkout option, this funtion is called and 
+   * it sets the selected payment method in the state and redirects to the next section
+   */
+  handleDefaultPaymentSelection(paymentMethod) {
+    const { nextSection, handleSetPaymentMethod } = this.props
+    handleSetPaymentMethod(paymentMethod)
+    nextSection()
+  }
+
+  render () {
+    const {
+      paypalCreateOrder,
+      paypalOnApprove,
+      handleSetPaymentMethod
+    } = this.props
+
+    return (
+      <div aria-label='Payment method' className='o-form c-payment-methods'>
+        <this.PaymentMethodHeader title={'Payment Method'} />
+        <div className='c-payment-methods__options'>
+          <this.PayPalButton 
+            paypalCreateOrder={paypalCreateOrder}
+            paypalOnApprove={paypalOnApprove}
+            handleSetPaymentMethod={handleSetPaymentMethod}
+          />
+          <h4 className='c-payment-methods__option-text'>OR</h4>
```

Comment:
Nitpicking, but I would use a regular `p` tag instead of `h4`, especially if h1 through to h3 are not present. `h4` is supposed to semantically represent a header, I presume you used it here because its styling happened to look good but we shouldn't rely on that. A client might want to customise all headers and will get unintended results at checkout.